src/sync.ts

```diff
@@ -343,35 +303,41 @@ export class Sync {
             }
           })
         ) {
+          // Gist files are the same as the local files.
           if (!localConfig.extConfig.forceUpload) {
-            if (
-              state.context.globalState.get<boolean>(
-                "gistNewer.dontShowThisAgain"
-              )
-            ) {
-              return;
-            }
+            vscode.window.setStatusBarMessage(
+              localize("cmd.updateSettings.info.gotLatestVersion"),
+              5000
+            );
+            // Exit early to avoid unneeded upload.
+            return;
+          }
+          // Fall through to upload code for forced upload case.
+        } else {
+          // Gist files are different from the local files.
+          const gistNewer = await github.IsGistNewer(
+            syncSetting.gist,
+            customSettings.lastDownload
+          );
+          if (gistNewer && !localConfig.extConfig.forceUpload) {
+            // Last local download is prior to the last gist upload, so
+            // the local settings may be out of date.
             const message = await vscode.window.showInformationMessage(
               localize("common.prompt.gistNewer"),
-              "Yes",
-              "Don't Show This Again"
+              localize("common.button.yes"),
+              localize("common.button.no"),
```

Comment:
is `,`  required at the end ?

Reply:
I've fixed this in: https://github.com/shanalikhan/code-settings-sync/pull/1042/commits/8125430f87bbb68a85179c5a7765370bff9a944c.  Thanks.